### DISCUSSION TITLE
Docs not explicit on the Option Type?

### DISCUSSION THREAD

**Discussion by pweeshe** [JedWatson/react-select#5060] [General]
The Docs, [here](https://react-select.com/props#prop-types), describe the OptionType as `type OptionType = { [string]: any }`,
This doesn't seem very helpful or clear about the option properties it could take?  It'd be nice to have them around here.

**Comment by Rall3n** [comment]
Hello @pweeshe,

the typings in the documentation for `OptionType` are as explicit as it gets. Because of the props `getOptionValue`, `getOptionLabel` and `isOptionDisabled` you can use any object shape you want for an option.
You are not limited to the by default expected shape of `{ value, label }`.

Examples for the usage of these props can be found [in the documentation](https://react-select.com/advanced#replacing-builtins).
